### DISCUSSION TITLE
PR Title

### DISCUSSION THREAD

**Discussion by umitcan07** [bounswe/bounswe2024group11#228] [General]
Hi team,
Keep in mind that when we merge a Pull Request, the branch we merge into will have a commit with the commit message equal to the PR title. So, let's use the same conventions on the PR titles as commit messages.

P.S. When we squash & merge a PR, the individual commits in the relevant branch will not make their way to the main.

**Comment by MucahitErdoganUnlu** [comment]
If so, does it make sense to push per commit?
